src/modules/validatorProfile/components/publicGood/ValidatorInfrastructure.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="block overflow-x-auto w-full p-0">
+    <div class='title'>RPC / LCD nodes:</div>
+    <el-table :data="nodesData" class="is-light text-white">
+      <el-table-column label="Network" min-width="33%">
+        <template #default="scope">
+          <span class="flex items-center">
+            <font-awesome-icon :icon="['fas', 'square']" :style="{ color: scope.row.status ? '#56dd1d' : '#DD1D1DFF' }" />
+            {{scope.row.network}}
+          </span>
+        </template>
+      </el-table-column>
+      <el-table-column label="Endpoint" min-width="33%">
+        <template #default="scope">
+          <div class="flex items-center">
+            {{scope.row.endpoint}}
+          </div>
+        </template>
+      </el-table-column>
+      <el-table-column label="Endpoint" min-width="33%">
+        <template #default="scope">
+          <div class="flex items-center">
+            {{scope.row.moniker}}
+          </div>
+        </template>
+      </el-table-column>
+    </el-table>
+  </div>
+
+  <div class="block overflow-x-auto w-full p-0">
+    <div class='title'>Relayer nodes:</div>
+    <el-table :data="relayersData" class="is-light text-white">
+      <el-table-column label="Network" min-width="33%">
+        <template #default="scope">
+          <span class="flex items-center">
+            <font-awesome-icon :icon="['fas', 'square']" :style="{ color: scope.row.status ? '#56dd1d' : '#DD1D1DFF' }" />
+            {{scope.row.network}}
+          </span>
+        </template>
+      </el-table-column>
+      <el-table-column label="Channel" min-width="33%">
+        <template #default="scope">
+          <div class="flex items-center">
+            {{scope.row.channel}}
+          </div>
+        </template>
+      </el-table-column>
+      <el-table-column label="Endpoint" min-width="33%">
+        <template #default="scope">
+          <div class="flex items-center">
+            {{scope.row.moniker}}
+          </div>
+        </template>
+      </el-table-column>
+    </el-table>
+  </div>
+
+  <div class="block overflow-x-auto w-full p-0">
+    <div class='title'>Archive nodes</div>
+    <el-table :data="archiveNodesData" class="is-light text-white">
+      <el-table-column label="Network" min-width="33%">
+        <template #default="scope">
+          <span class="flex items-center">
+            <font-awesome-icon :icon="['fas', 'square']" :style="{ color: scope.row.status ? '#56dd1d' : '#DD1D1DFF' }" />
```

Comment:
`'#56dd1d' : '#DD1D1DFF'`

i suggest better to use color constants like apple and lust for config

Reply:
and may be would better to create function for this condition. because this logic is used in many place 

what do you think?